development-docs/DEV_GUIDE.md

```diff
@@ -19,209 +25,286 @@ This document gives a detailed breakdown of the various build processes and opti
+
+   By default, the `docker_push` target will build the images under the strimzi organisation (
+   e.g. `strimzi/operator:latest`) and attempt to push them to the strimzi repositories on the Docker Hub. Only certain
+   users are approved to do this, so you should push to your own Docker Hub organisation (account) instead. To do this,
+   make sure that the `DOCKER_ORG` and `DOCKER_REGISTRY` environment variables are set to the same value as your
+   username on the Docker Registry, and Docker Registry you are using before running the `make` commands.\
+   \
+   When the Docker images are build, they will be labeled in the
+   form: `docker_registry_name/docker_hub_username/operator:latest` in your local repository and pushed to your Docker
+   Hub account under the same label.
 
 3. Now build the Docker images and push them to your repository on Docker Hub:
 
         make clean
         make all
 
-   Once this completes you should have several new repositories under your Docker Hub account (`docker_registry_name/docker_hub_username/operator`, `docker_registry_name/docker_hub_username/kafka` and `docker_registry_name/docker_hub_username/test-client`).
+   Once this completes you should have several new repositories under your Docker Hub account:
+    - `docker_registry_name/docker_hub_username/jmxtrans`
+    - `docker_registry_name/docker_hub_username/kafka`
+    - `docker_registry_name/docker_hub_username/kaniko-executor`
+    - `docker_registry_name/docker_hub_username/operator`
+    - `docker_registry_name/docker_hub_username/test-client`
 
-   The tests run during the build can be skipped by setting the `MVN_ARGS` environment variable and passing that to the make command:
+The tests run during the build can be skipped by setting the `MVN_ARGS` environment variable and passing that to the
+make command:
 
         make clean
         make MVN_ARGS='-DskipTests -DskipITs' all
 
-4. To use the newly built images, update the `install/cluster-operator/060-Deployment-strimzi-cluster-operator.yml` to obtain the images from your repositories on Docker Registry rather than the official Strimzi images:
-
-    ```
-    sed -Ei -e "s#(image|value): quay.io/strimzi/([a-z0-9-]+):latest#\1: $DOCKER_REGISTRY/$DOCKER_ORG/\2:latest#" \
+4. To use the newly built images, update
+   the `packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml` to obtain the images from your
+   repositories on Docker Registry rather than the official Strimzi images:
+   \
+   \
+   **Linux**
+   ```
+   sed -Ei -e "s#(image|value): quay.io/strimzi/([a-z0-9-]+):latest#\1: $DOCKER_REGISTRY/$DOCKER_ORG/\2:latest#" \
             -e "s#(image|value): quay.io/strimzi/([a-zA-Z0-9-]+:[0-9.]+)#\1: $DOCKER_REGISTRY/$DOCKER_ORG/\2#" \
             -e "s#([0-9.]+)=quay.io/strimzi/([a-zA-Z0-9-]+:[a-zA-Z0-9.-]+-kafka-[0-9.]+)#\1=$DOCKER_REGISTRY/$DOCKER_ORG/\2#" \
-            install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml
```

Comment:
Without this line the commmand won't work?

Reply:
Good catch @tomncooper, will fix it up in my NZ morning :)

Reply:
But it will need to be updated to point to `packaging/`.